docs/src/description/variables.md

```diff
@@ -0,0 +1,44 @@
+# Local Variables and Built-in Variables
+This section describes the syntax for declaring local variables and built-in variables.
+
+## Variables
+
+In AirScript, variables can be declared in `boundary_constraints` and `integrity_constraints` sections and can contain any expression. Variables can be of three types:
```

Comment:
Maybe this should say "can contain any expression that would be valid within that source section" (Since referencing a periodic column in `boundary_constraints` is invalid, for example)

Also, since we currently can't use `boundary_variables` in the boundary constraints section, maybe we should change that? Although I do have the PR ready to drop :)

Reply:
Should we merge that PR before release?